steadybit/README.md

```diff
@@ -0,0 +1,66 @@
+# Steadybit
+
+## Overview
+
+[Steadybit][1] is a chaos-engineering platform allowing you to simulate turbulent conditions in a controlled way, to improve system reliability and guide your organization to better incident management.
+
+Through this integration, you can leverage Datadog information within chaos-engineering experiments and see chaos-engineering activity in the form of events within Datadog.
```

Comment:
Could you provide 1-2 sentences on what customers will get out of this integration (the value)? Typically we like to have integrations include:

1. What does Steadybit do in 1-2 sentences?
2. What will customers get from this integration (the value)? 
3. What types of data is being sent to datadog? 

For context, we want customers to be able to read the overview and know exactly what the integration and partner does :)

Reply:
Hey @emarsha94,

would the following be a good replacement for the `Through this integration` paragraph that satisfies points 2) and 3)?

> Through this integration, you can leverage Datadog information within chaos engineering experiments. For example, to verify that Datadog monitors report an *Ok* status. Chaos engineering activity is also reflected within Datadog in the form of events. You can use these events in various ways, e.g., as event overlays to communicate chaos engineering activity to your teams.

Reply:
It's a great start! A couple of questions: 
1. What type of information would customers leverage?
2. What do you mean by experiments? 
3. What type of chaos engineering activity is sent? What insight does this provide?

Reply:
> What type of information would customers leverage?

Datadog monitors' status. The status could be used, for example, as a precondition for an experiment. Meaning: Only start the attack on a system when the system is not already facing any issues, i.e., `monitor status == Ok`.

> What do you mean by experiments?

Experiments are a core term within the chaos engineering space. Users finding this integration relevant should already be aware of the term. For more information you can read the [Principles of Chaos Engineering](https://principlesofchaos.org/)

> What type of chaos engineering activity is sent? What insight does this provide?

The information captured within the events transmitted to Datadog reflect:

 1. There an experiment started/ended
 2. The result of experiment
 3. In what environment the experiment was executed (which can be used to scope/filter events for Datadog event overlays)